iceoryx_binding_c/test/moduletests/test_event_info.cpp

```diff
@@ -78,10 +78,16 @@ class iox_event_info_test : public Test
     iox::mepoo::SharedChunk getChunkFromMemoryManager()
     {
         constexpr uint32_t PAYLOAD_SIZE{100U};
-        return m_memoryManager.getChunk(PAYLOAD_SIZE,
-                                        iox::CHUNK_DEFAULT_PAYLOAD_ALIGNMENT,
-                                        iox::CHUNK_NO_CUSTOM_HEADER_SIZE,
-                                        iox::CHUNK_NO_CUSTOM_HEADER_ALIGNMENT);
+
+        auto chunkSettingsResult = ChunkSettings::create(PAYLOAD_SIZE, iox::CHUNK_DEFAULT_PAYLOAD_ALIGNMENT);
+        EXPECT_FALSE(chunkSettingsResult.has_error());
+        if (chunkSettingsResult.has_error())
+        {
+            return nullptr;
+        }
```

Comment:
unfortunately I couldn't use `ASSERT_THAT` since it would return with `void`. Preciously, there was just a `terminate` in the code when wrong parameter were specified.

Reply:
Preciously ... Gollum...

If I understand correctly it is because you have to return something here even in the error case. That is ok, since the test will fail due to the ``EXPECT_THAT``

Reply:
Exactly. In this case an empty `SharedChunk` will be created and depending on the test this might lead to a termination. I could also just terminate in this case but it should never happen since the `ChunkSettings` parameter are valid.